viite-UI/src/less/site/project-change-table.less

```diff
@@ -42,22 +42,22 @@
   margin-top: 6px;
 }
 .change-table {
-  width: 1116px; //1132px
   display: block;
   height: inherit;
 }
-.change-table-dimensions {
-  width:1115px;
-}
 .project-change-table-dimension {
-  width: 66px;
+  width: 54.3px;
+  max-width: 60.3px;
   color: #5a5a57;
   font-weight: bold;
   height: 28px;
   font-size: 12px;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
 }
 .project-change-table-dimension-first {
-  width: 133px;
+  width: 128px;
```

Comment:
Prefer "em" as the measurement on text containers. For pictures, margins etc. px is good but font size should define the table size.